vercel/package.json

```diff
@@ -2,28 +2,26 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build": "react-router build && node vercel/prepare.js",
-    "dev": "cross-env NODE_ENV=development node ./dev-server.js",
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
```

Comment:
Start script is broken because build/server contains another folder `nodejs-eyJydW50aW1lIjoibm9kZWpzIn0` or a few per route.